docs/migrate_from_active_model_serializers.md

```diff
@@ -0,0 +1,357 @@
+---
+title: Upgrading from ActiveModelSerializers
+---
+
+<!-- @format -->
+
+This guide is aimed at helping ActiveModelSerializers users transition to Alba, and it consists of three parts:
+
+1. Basic serialization
+2. Complex serialization
+3. Unsupported features
+
+## Example class
+
+TODO:
+Example clsss is included `ActiveModel::Model`, because [serializing PORO with AMS is pretty hard](https://github.com/rails-api/active_model_serializers/blob/0-10-stable/docs/howto/serialize_poro.md).This example also works with ActiveRecord.
+
+```rb
+class User
```

Comment:
@okuramasafumi 
I can't find how to serialize with object included ActiveModel::Model.
https://github.com/rails-api/active_model_serializers/tree/0-10-stable#high-level-behavior
`SomeResource < ActiveModelSerializers::Model` is a bad example of both compatibility, so I'd like to use `SomeResource < ActiveRecord::Base`, but is that okay?
Like this.
```rb
class User < ActiveRecord::Base
  # columns: id, created_at, updated_at
  has_one :profile
  has_many :articles
end
```

Reply:
Sorry for the late reply.
Yes, using `AR::Base` is fine.